packages/dataviews/src/dataviews-layouts/timeline/style.scss

```diff
@@ -0,0 +1,189 @@
+@use "@wordpress/base-styles/colors" as *;
+@use "@wordpress/base-styles/variables" as *;
+
+.dataviews-view-timeline {
+	margin: 0 0 auto;
+	padding: $grid-unit-10 $grid-unit-60;
+
+	.dataviews-view-timeline__group-header {
+		font-size: $font-size-large;
+		font-weight: $font-weight-medium;
+		color: $gray-600;
+		margin: 0 0 $grid-unit-10 0;
+		padding: 0;
+	}
+
+	.dataviews-view-timeline__item {
+		margin: 0;
+
+		.dataviews-view-timeline__row {
+			position: relative;
+			padding: 0;
+			box-sizing: border-box;
+
+			.dataviews-view-timeline__dropdown-cell {
+				position: relative;
+				z-index: 10;
+				min-width: 32px;
+
+				.components-button {
+					position: relative;
+					z-index: 1;
+				}
+			}
+		}
+	}
+
+	.dataviews-view-timeline__event-content {
+		padding: $grid-unit-15 0;
+		min-height: $grid-unit-05 * 13;
+		flex-grow: 1;
+		gap: $grid-unit-10;
+
+		.dataviews-view-timeline__event-title {
+			position: relative;
+			display: flex;
+			align-items: center;
+			flex: 1;
+			min-height: $grid-unit-30;
+			line-height: $grid-unit-30;
+			overflow: hidden;
+			font-weight: 600;
+			font-size: $font-size-medium;
+			color: $gray-900;
+
+			&--clickable {
+				cursor: pointer;
+
+				&:focus-visible {
+					outline: 2px solid var(--wp-admin-theme-color);
+					outline-offset: 2px;
+					border-radius: $radius-small;
+				}
+			}
+		}
+
+		.dataviews-view-timeline__primary-actions {
+			display: flex;
+			width: max-content;
+			flex: 0 0 auto;
+
+			.components-button {
+				position: relative;
+				z-index: 1;
+			}
+		}
+
+		.dataviews-view-timeline__fields {
+			color: $gray-700;
+			display: flex;
+			gap: $grid-unit-15;
+			row-gap: $grid-unit-05;
+			flex-wrap: wrap;
+			font-size: $font-size-medium;
+
+			&:empty {
+				display: none;
+			}
+
+			.dataviews-view-timeline__field {
+				&:has(.dataviews-view-timeline__field-value:empty) {
+					display: none;
+				}
+			}
+
+			.dataviews-view-timeline__field-value {
+				min-height: $grid-unit-30;
+				line-height: $grid-unit-05 * 5;
```

Comment:
Think there's a var for this value.

Reply:
Thanks https://github.com/WordPress/gutenberg/pull/72780/commits/41fc54c30771898731e6fb8ac43ebf909072b267